### ISSUE TITLE
outline benchmark coverage per platform in README

### ISSUE THREAD

**Issue by basepi** [hubblestack/hubble#75]
_From @basepi on March 7, 2017 22:49_

_From @cedwards on April 25, 2016 3:48_

We need to somehow track what percentage coverage we have for a platform for each major benchmark. Example: Ubuntu-14.04 CIS: 80%, RHEL-6 CIS: 40%, etc.

_Copied from original issue: hubblestack/nova#60_

_Copied from original issue: hubblestack/hubble-salt#31_

**Comment by basepi** [comment]
Updated title. This needs to be done but will be painful and is very low priority in my mind.

**Comment by gjones623** [comment]
https://jira.corp.adobe.com/browse/CSTTEAM-44044

**Comment by alias454** [comment]
I was working on figuring out this problem myself so I can make sure the audits are as up to date as they can be. This generated some data that you might be interested in. My take used the latest benchmarks for a given platform and then checked the current audit checks against that. I did notice the ubuntu 14.04 LTS audit show almost 100% mising but I think that is related ot he control sections changing.

![screenshot from 2019-01-27 16-44-31](https://user-images.githubusercontent.com/10768415/51808879-d8708100-225f-11e9-8853-45ab8f4d5558.png)

These are .csv files but github requires a .txt extension
[Missing_Red_Hat_Enterprise_Linux_7_Benchmark_v220.txt](https://github.com/hubblestack/hubble/files/2800954/Missing_Red_Hat_Enterprise_Linux_7_Benchmark_v220.txt)
[Missing_Red_Hat_Enterprise_Linux_6_Benchmark_v210.txt](https://github.com/hubblestack/hubble/files/2800955/Missing_Red_Hat_Enterprise_Linux_6_Benchmark_v210.txt)
[Missing_Red_Hat_Enterprise_Linux_5_Benchmark_v220.txt](https://github.com/hubblestack/hubble/files/2800956/Missing_Red_Hat_Enterprise_Linux_5_Benchmark_v220.txt)
[Missing_Microsoft_Windows_Server_2016_RTM_Release_1607_Benchmark_v100.txt](https://github.com/hubblestack/hubble/files/2800957/Missing_Microsoft_Windows_Server_2016_RTM_Release_1607_Benchmark_v100.txt)
[Missing_Microsoft_Windows_Server_2012_R2_Benchmark_v230.txt](https://github.com/hubblestack/hubble/files/2800958/Missing_Microsoft_Windows_Server_2012_R2_Benchmark_v230.txt)
[Missing_Microsoft_Windows_Server_2008_R2_Benchmark_v310.txt](https://github.com/hubblestack/hubble/files/2800959/Missing_Microsoft_Windows_Server_2008_R2_Benchmark_v310.txt)
[Missing_Distribution_Independent_Linux_Benchmark_v110.txt](https://github.com/hubblestack/hubble/files/2800960/Missing_Distribution_Independent_Linux_Benchmark_v110.txt)
[Missing_Debian_Linux_9_Benchmark_v100.txt](https://github.com/hubblestack/hubble/files/2800961/Missing_Debian_Linux_9_Benchmark_v100.txt)
[Missing_Debian_Linux_8_Benchmark_v200.txt](https://github.com/hubblestack/hubble/files/2800962/Missing_Debian_Linux_8_Benchmark_v200.txt)
[Missing_Debian_Linux_7_Benchmark_v100.txt](https://github.com/hubblestack/hubble/files/2800963/Missing_Debian_Linux_7_Benchmark_v100.txt)
[Missing_CentOS_Linux_7_Benchmark_v220.txt](https://github.com/hubblestack/hubble/files/2800964/Missing_CentOS_Linux_7_Benchmark_v220.txt)
[Missing_CentOS_Linux_6_Benchmark_v210.txt](https://github.com/hubblestack/hubble/files/2800965/Missing_CentOS_Linux_6_Benchmark_v210.txt)
[Missing_Amazon_Linux_Benchmark_v210.txt](https://github.com/hubblestack/hubble/files/2800966/Missing_Amazon_Linux_Benchmark_v210.txt)
[Benchmark_Reporting_Totals.txt](https://github.com/hubblestack/hubble/files/2800967/Benchmark_Reporting_Totals.txt)
[Missing_Ubuntu_Linux_1804_LTS_Benchmark_v100.txt](https://github.com/hubblestack/hubble/files/2800968/Missing_Ubuntu_Linux_1804_LTS_Benchmark_v100.txt)
[Missing_Ubuntu_Linux_1604_LTS_Benchmark_v110.txt](https://github.com/hubblestack/hubble/files/2800969/Missing_Ubuntu_Linux_1604_LTS_Benchmark_v110.txt)
[Missing_Ubuntu_Linux_1404_LTS_Benchmark_v210.txt](https://github.com/hubblestack/hubble/files/2800970/Missing_Ubuntu_Linux_1404_LTS_Benchmark_v210.txt)
[Missing_Ubuntu_1204_LTS_Server_Benchmark_v110.txt](https://github.com/hubblestack/hubble/files/2800971/Missing_Ubuntu_1204_LTS_Server_Benchmark_v110.txt)

**Comment by basepi** [comment]
This is good work! I doubt our team at Adobe will be able to spend much time on these this year, but once I rewrite nova to fix the format (planned for this year) and we get past the python3 migration and whatnot, I'm hoping we can dedicate some time to truing these up.

**Comment by alias454** [comment]
Since a rewrite will take some time and you mentioned the Adobe team won't get to truing up the audits for awhile, will you take some PR's as I add new benchmarks? I was planning on adding the latest version benchmark checks as new audit files, as I work on them for my own needs.

**Comment by basepi** [comment]
Yes, we would still love additions. I'll be programmatically converting to the new format so your work will not be wasted. I just wanted to be up-front about our priorities. :)
